bin/builder.py

```diff
@@ -39,6 +39,7 @@ def ifind_files(patterns):
 
 
 def build(target_dir="dist", release="dev"):
+    os.chdir(PROJECT_ROOT)

```

Comment:
@guillermooo  `python builder.py` is run in a subshell, so after running it the current directory will not be changed.